--- src/rust/src/lib.rs	2024-02-24 01:04:55.000000000 +0000
+++ src/rust/src/lib.rs	2024-06-19 15:14:49.650689000 +0000
@@ -46,7 +46,7 @@
     // any time soon.
     let load_legacy = env::var("CRYPTOGRAPHY_OPENSSL_NO_LEGACY")
         .map(|v| v.is_empty() || v == "0")
-        .unwrap_or(true);
+        .unwrap_or(false);
     let legacy = if load_legacy {
         let legacy_result = provider::Provider::load(None, "legacy");
         _legacy_provider_error(legacy_result.is_ok())?;
